features/323-login-id/stage-1.md

```diff
@@ -0,0 +1,91 @@
+# Implict login ID key, Login ID realm
+
+## Database Schema
+
+- In `_auth_provider_password` table:
+    - A column `realm` (`TEXT NOT NULL`) would be added.
+    - The existing unique index on `login_id` would be changed to include `realm`.
+
+## Configuration
+
+- Add field `realmWhitelist` (list of strings, optional).
```

Comment:
I think we should be specific on weather empty string (or default) is part of the whitelist or not.

e.g. if
`realmWhitelist = ["doctor", "patient"]`

would Signup or Login without realm (or default) return error?

Reply:
Yes I missed the interpretation of missing whitelist, I'll fix it: it should be `(list of strings, optional, default to ["default"])`.

The signup/login without specifying realm case is specified in 'Auth Logic' section.